src/s2e/Plugins/OSMonitors/Linux/DecreeMonitor.cpp

```diff
@@ -234,7 +243,7 @@ klee::ref<klee::Expr> DecreeMonitor::readMemory8(S2EExecutionState *state, uint6
 uint64_t DecreeMonitor::getPid(S2EExecutionState *state, uint64_t pc) {
     target_ulong pid;
     target_ulong taskStructPtr = getTaskStructPtr(state);
-    unsigned pidAddress = taskStructPtr + 320; // Offsets can be found with: pahole vmlinux -C task_struct
+    target_ulong pidAddress = taskStructPtr + decree::TASK_STRUCT_PID_OFFSET;
```

Comment:
Ok. Currently I only added the kernel module to the vanilla Linux kernel. I'll have to add it to the Decree version to do this.